src/learn.py

```diff
@@ -43,9 +43,16 @@ def learn(subreddit=None):
                     if (
                         sub.subscribers > 100000
                     ):  # easier to get away with stuff on big subs
-                        log.info("found: " + str(sub.display_name))
-                        subok = True
-
+                        log.info("checking if subreddit is blacklisted")
+                        with open('src/banned_subreddits.txt') as blacklisted_subreddits:
```

Comment:
Okay check out this part of utils.py. https://github.com/MrPowerScripts/reddit-karma-farming-bot/blob/master/src/utils.py#L51

Do you see how we create a variable of the name at the top, and then we generate an empty variable list and then populate that list with the words from the file? That's a good practice to do with the disallowed list as well. Then we can use that variable in multiple places. which we'll need because this isn't the only place you need to check for the block list. there are other functions as well.

Reply:
Thanks for the tips, where else would be need to check for the blacklisted subs though ?

Reply:
So here when we do a random reply. https://github.com/MrPowerScripts/reddit-karma-farming-bot/blob/master/src/reddit.py#L351

and here in random submission: https://github.com/MrPowerScripts/reddit-karma-farming-bot/blob/master/src/reddit.py#L291

Those are also locations where we get a random subreddit to either posting or replying. Your location is for the bot learning. Ideally, we could figure out how to create a `filter_subreddits` function that we can use in each location. and then we would only have to edit that function one time when we want to adding filtering options like this. but currently the data isn't normalized in each location. so it's probably just easier to try and add the filtering logic to each section with whatever method is piratical. but I can help out with those parts when the global variable in utils.py